app/views/business_contact_forms/_form.slim

```diff
@@ -3,7 +3,7 @@
     = t('.title')
   .row.justify-content-center
     .col-md-8.col-11
-      = simple_form_for @business_contact_form do |f|
+      = simple_form_for @business_contact_form, html: { id: 'contact_form' } do |f|
```

Comment:
Doesn't the form have its own id? Just asking.

Reply:
It has its own id. Maybe I think in wrong direction, but for other contact forms we have id `contact-form`. I added the same id if in the future we want to add phone number in other contact forms. Then we will use the same intl-tel-input.scss. Maybe this is wrong thinking, over-something.